tapscript/script_test.go

```diff
@@ -0,0 +1,25 @@
+package tapscript
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestNewFundingScriptTree tests the NewFundingScriptTree function. We use this
+// function to filter out assets used for custom channel funding from the
+// balance queries. Therefor the script key cannot change since this would lead
+// to inconsistencies in the balance queries.
+func TestNewFundingScriptTree(t *testing.T) {
+	expectedTaprootKeyHex := "0350aaeb166f4234650d84a2d8a130987aeaf695020" +
```

Comment:
See my previous comment, but this key, threaded through `asset.NewScriptKey()` would start with `02`. Not sure if it makes sense to change this code after all, but perhaps a comment would be useful?